exercises/linked-list/instructions.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+Your task is to write a prototype for managing train routes.
```

Comment:
I'm not sure we need to repeat this here (this is more of a general comment I guess).

I think instead the task summary in `instructions` should be more technical. So for example:
> Your task is to build the prototype using doubly linked lists.

As these get concatenated into a single README, we can be sure people will always see the introduction.

Reply:
Yeah, I 100% agree, now that I understand the distinction. I've removed the sentence entirely, and I think the rest flows just fine without any other changes.